### ISSUE TITLE
Error running VG

### ISSUE THREAD

**Issue by hgz2021** [vgteam/vg#3365]
Hello, I installed the latest version of VG with Conda. The installation went well, but there was a problem typing vg -h 
FATAL: kernel too old
Aborted (core dumped)

**Comment by adamnovak** [comment]
How old is your kernel? What does `uname -a` say about it?

The static binaries we distribute on the [releases page](https://github.com/vgteam/vg/releases) *should* run on Linux 3.2 or later, which is from around 2012, although we don't actually test that far back. I believe that Conda builds their own binaries, though, and I can't vouch for what Linux versions they support.

You should be able to use `file $(which vg)` to inspect the binary, and it will tell you what version of Linux it ought to work on. Our releases look like:

```
vg: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, BuildID[sha1]=556f81232dabefed5605f53ec9f47cb6ecacf881, for GNU/Linux 3.2.0, not stripped
```

**Comment by jingydz** [comment]
## Command
vg]$ ./vg --version
FATAL: kernel too old
Aborted (core dumped)

vg]$ file $(which ./vg)
/xxx/software/VG/vg/vg: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, for GNU/Linux 3.2.0, not stripped

vg]$ uname -a
Linux admin 2.6.32-504.el6.x86_64 #1 SMP Wed Oct 15 04:27:16 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

Is it because my Linux kernel version is too old?

**Comment by faithokamoto** [comment]
Yeah, probably. Also we've had a lot of vg version / build updates since this, including many dependency changes. Closing as stale.
